data_diff/database.py

```diff
@@ -714,6 +716,65 @@ def normalize_value_by_type(self, value: str, coltype: ColType) -> str:
 HELP_PRESTO_URI_FORMAT = "presto://<user>@<host>/<catalog>/<schema>"
 
 
+@dataclass
+class MatchUriPath:
+    database_cls: type
+    params: List[str]
+    kwparams: List[str] = []
+    help_str: str
+
+    def match_path(self, dsn):
+        dsn_dict = dict(dsn.query)
+        matches = {}
+        for param, arg in zip_longest(self.params, dsn.paths):
+            if param is None:
```

Comment:
What is it's empty, or does that become None in the DSN parser?

Reply:
What's empty?

Reply:
If I pass `//database`

Reply:
kk